docs/contributing/pull-requests/code-review.md

```diff
@@ -46,20 +55,36 @@ requirements, which may not necessarily be in line with the previous solution.
 
 Please use the review statuses appropriately.
 
+:::tip Think about your audience
+
+When providing comments or requesting changes, keep in mind the experience level of the author.
+
+If the engineer is new to the company and the codebase or less experienced overall, they would
+probably welcome more explicit background on your concerns and more concrete suggestions, whereas a
+more experienced engineer would prefer general guidance so that they can solve the problem
+themselves.
+
+:::
+
 #### Comment
 
 Comment is a great way to discuss things without explicitly approving or requesting changes.
 
+We use [conventional comments](https://conventionalcomments.org) to help convey the intent and
+expected action taken from each comment on the PR.
+
 #### Request changes
 
 Request changes should be used when you believe something **needs** to change prior to the PR
 getting merged, as it will prevent someone else from approving the PR before your concerns have been
 tackled.
 
-We shouldn’t hesitate to use this status, however we should give clear feedback on what needs to
-change for the PR to get approved. Likewise a PR author should not be discouraged by a _request for
-changes_, it's simply an indication that changes should be made prior to the PR being merged. This
-is common.
+We shouldn’t hesitate to use this status. However, it does come with obligations for the reviewer.
+By blocking the PR from progressing, they have taken an ownership stake in it and should make
+themselves available to answer clarifying questions. They should also give clear feedback on what
+needs to change for the PR to get approved. Likewise, a PR author should not be discouraged by a
+request for changes, it's simply an indication that changes should be made prior to the PR being
+merged. This is common.
```

Comment:
I would argue that every review comes with this obligations. I'm worried this might associate request changes negatively.

Reply:
Can you elaborate? Not sure why this language change does anything more than expect the reviewer to stay engaged.

I too want to be sure "request changes" is taken very objectively.

Reply:
"However, it does come with obligations for the reviewer. By blocking the PR from progressing, they have taken an ownership stake in it and should make themselves available to answer clarifying questions. They should also give clear feedback on what needs to change for the PR to get approved."

Leaving any review usually comes with this obligations. I would prefer to see this lifted into some general section. I do think we can highlight that request changes blocks the PR from moving.

Reply:
I tried to bridge this distinction with the changes I just pushed - I added a section at the top of the "Reviewing" section that applies to all types of review, and changed this section to indicate that "additional responsibility" is implied by blocking the PR.  Let me know how that looks.